src/WORKSPACE

```diff
@@ -16,6 +16,17 @@ load("@rules_foreign_cc//foreign_cc:repositories.bzl", "rules_foreign_cc_depende
 # https://bazelbuild.github.io/rules_foreign_cc/0.5.1/flatten.html#rules_foreign_cc_dependencies
 rules_foreign_cc_dependencies()
 
+rules_python_version = "740825b7f74930c62f44af95c9a4c1bd428d2c53"  # Latest @ 2021-06-23
```

Comment:
We seem to be able to get newer versions of `rules_python` which support Bazel 4+. Haven't really looked into their changes, but it may be useful later on.

Reply:
Upgraded to the most recent version